### PR TITLE
Label input (Issue 65)

### PR DESCRIPTION
**Pull request by Tanvir-rahman**
## What is the Purpose?
Added Labeled input with required UI and disabled props as per the request.
https://github.com/hikaya-io/vue-ui-components/issues/65

## What was the approach?
Added another component as 'LabelInput'.
Added props 'labelPosition', 'required', 'disabled'
labelPosition can be set as 'top' / 'left'

## Are there any concerns to addressed further before or after merging this PR?
No

## Mentions?
@andrewtpham @michaelbukachi 

## Issue(s) affected?
Empty

### CODE REVIEW

**Comment by Tanvir-rahman** [comment]
Hello @michaelbukachi . I was thinking to solve this issue with all Input elements with this PR https://github.com/hikaya-io/vue-ui-components/issues/71 as improvement part :)

**Comment by michaelbukachi** [comment]
Wait. This PR is linked to which issue?

**Comment by ninetteadhikari** [comment]
> Hello @michaelbukachi . I was thinking to solve this issue with all Input elements with this PR #71 as improvement part :)

Hey @Tanvir-rahman agreed that there's a separate ticket for removing the `form` wrapper (#71) but I suggest we keep this PR clean and use #71 for cleaning up the mentioned components there.

File: src/stories/Input.stories.js [hunk 1]
```diff
@@ -62,3 +63,14 @@ export const inputLength = () => ({
     </div>
   `
 })
+
+export const labelInput = () => ({
+  components: { LabelInput },
+  template: `
+    <div>
+      <label-input placeholder="Input here" label="Label Input" labelPosition="top"></label-input>
+      <label-input placeholder="Input here" label="Required Input" labelPosition="left" required></label-input>
```
Comment (ninetteadhikari) [inline src/stories/Input.stories.js:72]: Can we please have them all with label on top? That is the template we are trying to go for

**Comment by Tanvir-rahman** [comment]
> > Hello @michaelbukachi . I was thinking to solve this issue with all Input elements with this PR #71 as improvement part :)
> 
> Hey @Tanvir-rahman agreed that there's a separate ticket for removing the `form` wrapper (#71) but I suggest we keep this PR clean and use #71 for cleaning up the mentioned components there.

Ok, as @michaelbukachi already mentioned we will loose the label because of removing parent form. So that's why at first we need to [introduce another component](https://github.com/hikaya-io/vue-ui-components/issues/71#issuecomment-623000993) for the parent form.

File: src/stories/Input.stories.js (hunk 1 above)
Comment (Tanvir-rahman) [reply src/stories/Input.stories.js:72]: Screenshot 2020-05-03 at 8 03 20 AM
The component has option to choose between left and top position. Do you want me to remove the left option availability?

File: src/components/input/LabelInput.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+</script>
+
+<style scoped>
+.label {
+  word-wrap: break-word;
+}
+.required {
+  color: red;
+  margin-right: 2px;
+}
```
[the remark this answers is not in the document]
Comment (Tanvir-rahman) [reply src/components/input/LabelInput.vue:80]: Screenshot 2020-05-03 at 8 39 08 AM

File: src/components/input/LabelInput.vue [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  }
+}
+</script>
+
+<style scoped>
+.label {
+  word-wrap: break-word;
+}
+.required {
+  color: red;
```
[the remark this answers is not in the document]
Comment (Tanvir-rahman) [reply src/components/input/LabelInput.vue]: Yep done 😃

File: src/stories/Input.stories.js (hunk 1 above)
Comment (ninetteadhikari) [reply src/stories/Input.stories.js:72]: ah you are right. No lets leave it open to choose the position of the label :)

**Comment by ninetteadhikari** [comment]
> > > Hello @michaelbukachi . I was thinking to solve this issue with all Input elements with this PR #71 as improvement part :)
> > 
> > 
> > Hey @Tanvir-rahman agreed that there's a separate ticket for removing the `form` wrapper (#71) but I suggest we keep this PR clean and use #71 for cleaning up the mentioned components there.
> 
> Ok, as @michaelbukachi already mentioned we will loose the label because of removing parent form. So that's why at first we need to [introduce another component](https://github.com/hikaya-io/vue-ui-components/issues/71#issuecomment-623000993) for the parent form.

Cool I created the form component issue as suggested #76 
So is this PR currently blocked by #76 then? Or what do you suggest to move forward?

**Comment by Tanvir-rahman** [comment]
> > > > Hello @michaelbukachi . I was thinking to solve this issue with all Input elements with this PR #71 as improvement part :)
> > > 
> > > 
> > > Hey @Tanvir-rahman agreed that there's a separate ticket for removing the `form` wrapper (#71) but I suggest we keep this PR clean and use #71 for cleaning up the mentioned components there.
> > 
> > 
> > Ok, as @michaelbukachi already mentioned we will loose the label because of removing parent form. So that's why at first we need to [introduce another component](https://github.com/hikaya-io/vue-ui-components/issues/71#issuecomment-623000993) for the parent form.
> 
> Cool I created the form component issue as suggested #76
> So is this PR currently blocked by #76 then? Or what do you suggest to move forward?

Yep. This is blocked currently. Will be working on issue 76 soon :)

**Comment by andrewtpham** [comment]
@Tanvir-rahman is this ready to be merged?

**Comment by Tanvir-rahman** [comment]
Hello. @andrewtpham. Nope there was some dependency of Form Element. It resolved. Will give final push by today after optimization.

**Comment by Tanvir-rahman** [comment]
Pr is ready for review. 
However, changed required field color to $red as did not find any associated with orange color.

File: src/components/input/LabelInput.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrewtpham) [reply src/components/input/LabelInput.vue:75]: Ive address the padding issue in #88 so we can resolve for now

**Pull request merged**